ui/src/components/Interapp.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
-import './App.css';
-import './main.css';
+import '../css/App.css';
+import '../css/main.css';
```

Comment:
Hmmm, I think `main.css` shouldn't be lurking any more. I was using it to fiddle around with the mockups, but it's gone from my code and files locally...